pkg/reconciler/events/auditlogs/resources/filterbuilder.go

```diff
@@ -23,15 +23,15 @@ type FilterBuilder struct {
 	resourceName string
 }
 
-func (fb *FilterBuilder) SetServiceName(serviceName string) {
+func (fb *FilterBuilder) WithServiceName(serviceName string) {
 	fb.serviceName = serviceName
```

Comment:
I meant to have something like:

```
func (fb *FilterBuilder) WithServiceName(name string) *FilterBuilder {
  fb.serviceName = serviceName
  return fb
}
```

Now you can chain these functions like:

```
fb = fb.WithServiceName("a").WithMethodName("b").WithResourceName("c")
```

Reply:
Done.